lms/djangoapps/grades/tasks.py

```diff
@@ -250,3 +267,21 @@ def _update_subsection_grades(course_key, scored_block_usage_key, only_if_higher
                     user=student,
                     subsection_grade=subsection_grade,
                 )
+
+
+def _course_task_args(course_key, **kwargs):
+    """
+    Helper function to generate tuples batched course task args based on course enrollment numbers.
```

Comment:
There are too many words in the middle of this sentence.  How about "Helper function to generate course-grade task args"?